.gitignore

```diff
@@ -9,6 +9,11 @@ npm-debug.*
 *.mobileprovision
 *.orig.*
 web-build/
+.expo-shared
+.idea
+.vscode
+app.json
+bable.config.js
```

Comment:
Why do we want to ignore these files from being committed?

Reply:
Sir, it was my mistake that I have set them in ignore file without knowing about them.
App.json contains configuration details about the project like author name, about project version which helps to understand the project. Previously app.json contains some security key but in new updates that part was removed.